python/kaitai/compress/__init__.py

```diff
@@ -1,5 +1,4 @@
-from .lz4 import Lz4
-from .zlib import Zlib
-from .lzma_raw import LzmaRaw
-from .lzma_lzma import LzmaLzma
-from .lzma_xz import LzmaXz
+from .core import *
+from .algorithms.zlib import Zlib
+from .algorithms.lzma import Lzma, LzmaLzma, LzmaRaw, LzmaXz
+from .algorithms.lz4 import Lz4
```

Comment:
This file / module naming makes it incompatible with all other implementations and totally kills the purpose of having equally named algorithm implementations in all languages.

Reply:
OK, I will remove them. They are there only for compatibility, but I guess we can break it.